### PR TITLE
2021-5g-chatbot-puppet

### PR DESCRIPTION
**Pull request by sqcn**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Comment by lijiarui** [comment]
Thanks for the blog！

Please make GitHub action green. You can run the `npm run test` locally to make this commit pass all of the tests first.

**Comment by lijiarui** [comment]
@sqcn Please make GitHub action green. You can run the npm run test locally to make this commit pass all of the tests first.

**Comment by sqcn** [comment]
@lijiarui Hi, we already fixed the last picture bug that CI Action mentioned. We just submitted but still yellow.

**Comment by sqcn** [comment]
@lijiarui  Hi, we have already fixed the last picture bug that CI Action mentioned. We just submitted but still yellow.

File: jekyll/_posts/2021-07-15-ospp-plan-5g-chatbot-puppet.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+“开源软件供应链点亮计划-暑期2021”（以下简称 暑期2021）是由中科院软件所与 openEuler 社区共同举办的一项面向高校学生的暑期活动。 旨在鼓励在校学生积极参与开源软件的开发维护，促进国内优秀开源软件社区的蓬勃发展。 根据项目的难易程度和完成情况，参与者还可获取“开源软件供应链点亮计划-暑期2021”活动奖金和奖杯。
+
+官网：<https://summer.iscas.ac.cn/#/homepage>
+
+本项目 [开发支持电信运营商5G Chatbot/RCS的 Wechaty接入Puppet模块] 系暑期2021支持的开源项目。
+
+## 开发支持电信运营商5G Chatbot/RCS的 Wechaty接入Puppet模块
+
+- 导师：李佳芮,康嘉
```
Comment (lijiarui) [inline jekyll/_posts/2021-07-15-ospp-plan-5g-chatbot-puppet.md]: Please add `李卓桓` here as well

File: jekyll/_posts/2021-07-15-ospp-plan-5g-chatbot-puppet.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    - 7/12 - 7/18
+    - 配置好服务器，测通相关接口
+  - 使用 <https://github.com/wechaty/wechaty-puppet-official-account> 项目作为模版，将其中的微信公众平台调用全部封装为RCS模块的调用
+    - 7/19 - 8/10
+    - 学习现有源码，学习RCS的相关知识。
+    - 接入wechaty，进行代码整合，提高代码质量。
+  - 项目中期总结
+    - 8/11 - 8/15
+    - 实现阶段性目标，总结项目阶段性成果。
+    - 无
```
Comment (lijiarui) [inline jekyll/_posts/2021-07-15-ospp-plan-5g-chatbot-puppet.md]: What do you mean `无` here, I suggest deleting unrelated intro in the blog.

File: jekyll/_posts/2021-07-15-ospp-plan-5g-chatbot-puppet.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+官网：<https://summer.iscas.ac.cn/#/homepage>
+
+本项目 [开发支持电信运营商5G Chatbot/RCS的 Wechaty接入Puppet模块] 系暑期2021支持的开源项目。
+
+## 开发支持电信运营商5G Chatbot/RCS的 Wechaty接入Puppet模块
+
+- 导师：李佳芮,康嘉
+
+- 学生：张瑞宁,邵琦
+
```
Comment (lijiarui) [inline jekyll/_posts/2021-07-15-ospp-plan-5g-chatbot-puppet.md:34]: Please add `项目介绍：https://github.com/wechaty/summer-of-wechaty/issues/74` here

**Review by lijiarui — CHANGES_REQUESTED** [review]
Thanks for the adjustment!

Please change as I request.

And it could be better if you can refer https://github.com/wechaty/wechaty.js.org/pull/1078/files to make the blog look better.

**Comment by lijiarui** [comment]
Thank you very much for your contribution!

## You are welcome to join [Wechaty Contributor Program](https://wechaty.js.org/docs/contributor-program/)

### 1. Join Wechaty Organization

> You've invited sqcn to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

I have invited you to join our Wechaty GitHub Organization, please accept it by following the above message.

### 2. Update Your Wechaty Contributor Profile

1. Please open [Contributor Hall of Fame](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0) and add yourself to the end of the list, so that other contributors will know you better!
1. Please add yourself to our Website [Contributors](https://wechaty.js.org/contributors/) by [creating a PR](https://github.com/wechaty/wechaty.js.org/tree/master/jekyll/_contributors) and refer to this PR link as well.

### 3. Join The Contributor Only WeChat Room

We also have a WeChat room for contributors only which can discuss Wechaty at a deeper level, you are welcome to join and if you are interested.

Please add my: wechat: _ruirui_0914_ and send this pr link to me. I will invite you into Wechaty Contributor Room

Link to https://github.com/wechaty/PMC/issues/16

Cheers!

**Pull request merged**